src/pd/cluster.rs

```diff
@@ -91,6 +93,18 @@ impl Cluster {
         req.safe_point = safepoint;
         req.send(&mut self.client, timeout).await
     }
+
+    pub async fn get_keyspace_id(&self, keyspace: &str) -> Result<u32> {
```

Comment:
There is a grpc method for this interface.

Reply:
I'll modify this part in the next PR.